### PR TITLE
Fix for aws s3 stream

### PR DESCRIPTION
**Pull request by alysonturing**

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/aws/actions/s3-stream-file/s3-stream-file.mjs [hunk 1]
```diff
@@ -18,14 +19,42 @@ export default {
...
+      await (new Promise((resolve) => {
+        const writeStream = fs.createWriteStream(tempLocalPath);
+        fileResponse.data.pipe(writeStream);
+        writeStream.on("finish", resolve);
+      }));
+      var stats = fs.statSync(tempLocalPath);
+      return {
+        stream: fs.createReadStream(tempLocalPath),
+        contentLength: stats.size,
+      };
```
Comment (GTFalcao) [inline components/aws/actions/s3-stream-file/s3-stream-file.mjs:42]: I suggest we stick to the `const` pattern here:
```suggestion
      const { size } = fs.statSync(tempLocalPath);
      return {
        stream: fs.createReadStream(tempLocalPath),
        contentLength: size,
      };
```

**Review by GTFalcao — COMMENTED** [review]
Hi @alysonturing , thanks for the fix, I've left just one small code comment!

**Review by GTFalcao — APPROVED** [review]
Looks good to me! 👏 
Moving to QA

**Pull request merged**